support-frontend/assets/helpers/abTests/__tests__/abtestTest.ts

```diff
@@ -927,3 +387,43 @@ it('targetPage matching', () => {
 		targetPageMatches('/uk/subscribe/digital/gift', nonGiftLandingNotAusNotUS),
 	).toEqual(false);
 });
+
+// ----- Helpers ----- //
+
+function buildVariant({ id = 'control' }: Partial<Variant>): Variant {
+	return { id };
+}
+
+function buildAudience({
+	offset = 0,
+	size = 1,
+	breakpoint,
+}: Partial<Audience>): Audience {
+	return { offset, size, breakpoint };
+}
+
+function buildTest({
+	variants = [buildVariant({})],
+	referrerControlled = false,
+	audiences = { ALL: buildAudience({}) },
+	isActive = true,
+	seed = 0,
+}: Partial<Test>): Test {
+	return {
+		variants,
+		audiences,
+		isActive,
+		referrerControlled,
+		seed,
+	};
+}
+
+function buildAcquisitionAbTest({
+	name = 't',
+	variant = 'control',
+}: Partial<AcquisitionABTest>): AcquisitionABTest {
+	return {
+		name,
+		variant,
+	};
+}
```

Comment:
These are the new `build` functions I added. The `Partial` utility type from typescript is really helpful here. It allows us to easily define functions that you can just provide with the arguments you care about for the test and have all the others fallback to sensible defaults.